hazelcast/src/test/java/com/hazelcast/quorum/map/MapReadWriteQuorumTest.java

```diff
@@ -455,35 +455,4 @@ public void testSubmmitToKeyOperationThrowsExceptionWhenQuorumSizeNotMet() throw
         foo.get();
     }
 
-    @Ignore
```

Comment:
Quorum for lock operations has never been implemented. I've left the tests if that's implemented in near future but it didn't happened. you can delete them.